app/com/linkedin/drelephant/tuning/hbt/PigHbtParameterRecommender.java

```diff
@@ -42,18 +42,30 @@
 
 public class PigHbtParameterRecommender {
   private List<AppResult> appResultList;
+  private HashMap<String, Double> maxHeuristicDetailValueMap = new HashMap();
   @Getter
   private Map<String, Double> jobSuggestedParameters = new HashMap<>();
   @Getter
   private Map<String, Double> latestAppliedParams = new HashMap<>();
   public Map<String, Double> essentialCounters = new HashMap<>();
 
+  public class MaxMemoryHeuristicsProperties {
```

Comment:
Why this class is public . Are u using it outside , if yes , then why its internal class

Reply:
I kept is public for Hive HBT if we do it sometime in future.